py/selenium/deprecated.py

```diff
@@ -0,0 +1,31 @@
+from warnings import warn
```

Comment:
Thanks @sandeepsuryaprasad will review more shortly, I put together a somewhat similar idea so i'm not too biased, open to just have a reusable mechanism to handle it in general.

one minor thing is new files should be prefixed with the licensing meta data

Reply:
> Thanks @sandeepsuryaprasad will review more shortly, I put together a somewhat similar idea so i'm not too biased, open to having just have a reusable mechanism to handle it in general that works well.
> 
> one minor thing is new files should be prefixed with the licensing meta data

Yes, I have added license stuff.